README.md

```diff
@@ -1,13 +1,32 @@
 ﻿# sgf.dev
 
+> Currently this project will only run in a Windows environment. We are using the Umbraco CMS which is built on the .NET Framework. We are keeping a sharp eye on the progress of their cross-platform release on .NET Core which will run on macOS and Linux.
+
 ## Setup
 - Copy `SgfDevs/config/connectionStrings.config.example` to `SgfDevs/config/connectionsStrings.config`.
     - Don't enter your connection string just yet. Umbraco will ask for that as a part of the install process and write that value to this file. If Umbraco sees that value is already set it will assume the database schema is already setup.
-- Open `SgfDevs/Web.config`, look for `<add key="Umbraco.Core.ConfigurationStatus" value="8.9.1" />`, and remove the `8.9.1`.
+- Open `SgfDevs/Web.config`, look for `<add key="Umbraco.Core.ConfigurationStatus" value="8.9.1" />`, and remove the `8.9.1` (or whatever version number we're currently on).
     - Umbraco will automatically add that value back as a part of the install process.
 - Run the project and open it in a browser. You should be presented with the Umbraco install screen.
-- Enter in the details for your local admin user and click Customize.
-- Enter in the connection details for your local database.
-- Login to the Umbraco backoffice and go to Settings > uSync.
+- Enter in the details for what will be your local admin user for the CMS and click **Customize**.
+
+![](https://mykebates.blob.core.windows.net/towk/sgfdevs/install_1.png)

```

Comment:
@mykebates Love the screenshots being added here and the updated instructions. Could we maybe just link off to these images instead of showing inline? I'm sure this README will get more and more added to it and these take a lot of space. Also, maybe we could drop these images into the repo locally instead of referencing them from outside the repo?

Reply:
Another option would be to have a SETUP.md file where we do keep the screenshots inline and we just link off to it from the main README. I'm fine with either one though

Reply:
From a slack convo between me and Myke. We'll merge in for now and look at a more robust setup wiki with a cleaner setup in the future when it's needed or when the current README gets more stuff added to it